spec/models/citation_spec.rb

```diff
@@ -0,0 +1,203 @@
+# == Schema Information
+#
+# Table name: citations
+#
+#  id           :integer          not null, primary key
+#  user_id      :integer
+#  citable_type :string
+#  citable_id   :integer
+#  source_url   :string
+#  type         :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+
+require 'spec_helper'
+
+def setup_for_x_scope_data
```

Comment:
Finding this setup hard to understand - any way this could be encapsulated in the factory?